runtime/common/src/claims.rs

```diff
@@ -47,6 +49,30 @@ pub trait Config: frame_system::Config {
 	type VestingSchedule: VestingSchedule<Self::AccountId, Moment=Self::BlockNumber>;
 	type Prefix: Get<&'static [u8]>;
 	type MoveClaimOrigin: EnsureOrigin<Self::Origin>;
+	type WeightInfo: WeightInfo;
+}
+
+pub trait WeightInfo {
+	fn claim() -> Weight;
+	fn validate_unsigned_claim() -> Weight;
+	fn mint_claim() -> Weight;
+	fn claim_attest() -> Weight;
+	fn validate_unsigned_claim_attest() -> Weight;
+	fn attest() -> Weight;
+	fn validate_prevalidate_attests() -> Weight;
+	fn move_claim() -> Weight;
+}
+
+pub struct TestWeightInfo;
+impl WeightInfo for TestWeightInfo {
+	fn claim() -> Weight { 0 }
+	fn validate_unsigned_claim() -> Weight { 0 }
+	fn mint_claim() -> Weight { 0 }
+	fn claim_attest() -> Weight { 0 }
+	fn validate_unsigned_claim_attest() -> Weight { 0 }
+	fn attest() -> Weight { 0 }
+	fn validate_prevalidate_attests() -> Weight { 0 }
+	fn move_claim() -> Weight { 0 }
 }
```

Comment:
Shouldn't that the generated by the bot inside the weights module?

Reply:
All that doesn't really work when these pallets are not their own crates.